drake/doc/developers.rst

```diff
@@ -43,6 +43,42 @@ available in the superbuild.
 class method runs as a unit test, and fails if it times out or calls
 ``error``. For more information, see ``drake/util/unitTest.m``.
 
+Supported Configurations
+========================
+
+The following table shows the configurations and platforms that Drake
+officially supports. Supported configurations are tested in continuous
+integration. All other configurations are provided on a best-effort basis.
+On Ubuntu and OS X, only makefile builds are supported.
```

Comment:
Thanks. Suggested text for last sentence:  "On Ubuntu and OS X, only CMake generators that emit makefile builds are supported."